### DISCUSSION TITLE
Creating gallery in the Jekyll theme from Adobe Lightroom

### DISCUSSION THREAD

**Discussion by ansiblejunky** [mmistakes/minimal-mistakes#4788] [Q&A]
Hi there, I use Adobe Lightroom for photography purposes and it allows export of web content with images but I'm not clear on how to get it into my Jekyll site. Would be amazing to retain the format of image numbers etc so my customers can select their favorite images. 

The file structure of the exported data
```
├── content
├── index.html
├── index_2.html
├── index_3.html
├── index_4.html
├── index_5.html
├── index_6.html
├── index_7.html
└── resources
```

**Comment by ansiblejunky** [comment]
This is another structure Lightroom exports to:
```
├── assets
│   ├── css
│   └── js
├── images
│   ├── large
│   └── thumbnails
└── index.html
```

**Reply by svaderia** [reply]
From what I understand Jekyll supports the `html` file, can you not just put the `index.html` into related folder and move the other assets to the related folder?
